code/unclassified/src/split_list/split_array.js

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I would remove this newline.

Reply:
How do I do that I'm new to this but need to learn.and need help please

Reply:
ask @arnavb he guided me a lot

Reply:
done !!!

Reply:
@danos7777 First learn how to use Github, and then pick and one of our top-level topics (data structures, online challenges, etc.) Pick a subtopic and work on it! When you open your PR, I will be there to review! Good luck! 👍

Reply:
If you are new, learn it from udacity 
https://in.udacity.com/course/how-to-use-git-and-github--ud775
ps: ignore `in`, its included because i am from India